frb_rust/src/misc/user_utils.rs

```diff
@@ -3,9 +3,6 @@ use crate::misc::panic_backtrace::PanicBacktrace;
 /// Setup defaults that is usually useful for a new project.
 /// Surely, you are free to customize everything.
 pub fn setup_default_user_utils() {
-    // setup log before others, such that we can see logs in other setup functions
-    #[cfg(feature = "log")]
-    setup_log_to_console();
```

Comment:
I agree to avoid breaking changes ... but are you sure about this one?

Here is the code
```Rust
#[cfg(feature = "log")]
fn setup_log_to_console() {
    #[cfg(target_os = "android")]
    let _ = android_logger::init_once(
        android_logger::Config::default().with_max_level(log::LevelFilter::Trace),
    );

    #[cfg(any(target_os = "ios", target_os = "macos"))]
    let _ = oslog::OsLogger::new("frb_user")
        .level_filter(log::LevelFilter::Trace)
        .init();

    #[cfg(wasm)]
    let _ = crate::misc::web_utils::WebConsoleLogger::init();

    // TODO add more platforms
}
```
It was called only indirectly, through setup_default_user_utils() - and I think it was added only in v.2.3 or v.2.4? I roughly remember that it broke my code - so removing might actually un-break code :)

Nevertheless, it just sets the log value to TRACE (I set INFO as a default) and initializes the mobile loggers.

So, the user's code would still compile. He would still see the log output, just additionally in stdout (as I assume, I might be wrong here). I can change the default log level to TRACE - but I think INFO is a better default?

Reply:
> and I think it was added only in v.2.3 or v.2.4

Oh you are right, this is a very special case. But if they are in that version, then we break (unbreak) them again! I am not sure whether this is the correct way...

(INFO lgtm)

Reply:
As always - it is ultimately your decision :)

I am thinking that these mobile logging crates might conflict with some Dart/Flutter packages.
I think that it would be better to keep the default logging simple.
Sure, one can change/not use it, if one has conflicts ...

But I wonder how breaking the change would actually be?
Unless I overlook something, it should really not be much noticeable to the user.

Meaning, there should be no compile error.
If I am wrong about stdout being automatically logged in the OsLogger & Android logger output the user would not see any difference.

Reply:
> stdout being automatically logged in the OsLogger & Android logger

Hmm IIRC long ago people saying that `println` will *not* print anything to android or ios outputs. (Maybe they are talking about another output?)

Except for the breaking change issue, looks like this removal is pretty reasonable.
What about this:

1. We remove this (just like what you have done)
2. In migration doc, we tell users what happens and what should be done (e.g. "please add the new `setup_our_good_logging_system!()` macro")

(TODO, finish this later)